### PR TITLE
WP-35 Implement drawer nav menu

### PR DESCRIPTION
**Pull request by ericgmoon**
image

On hover:
image

Automatic highlights based on the current path in the browser address bar:
image

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -22,6 +22,7 @@ const App = () => (
           <Route path="/signin" element={<SignInPage />} />
           <Route path="/signup" element={<SignUpPage />} />
           <Route path="/" element={<HomePage />} />
+          <Route path="/notes" element={<HomePage />} />
```
Comment (ericgmoon) [inline src/App.tsx:25]: This is a temporary route to allow us to test how the menu items highlight.

File: src/theme.tsx [hunk 2]
```diff
@@ -54,6 +54,12 @@ const theme = {
       light: '#5982c5',
       contrastText: '#fff',
     },
+    highlight: {
```
Comment (ericgmoon) [inline src/theme.tsx:57]: this is the redish colour used for the menu headers

**Comment by ericgmoon** [comment]
I know you said something about creating the links via `theme.tsx`? I forgot exactly what you meant, so have implemented it using `onClick` for now.

**Comment by ericgmoon** [comment]
Update: now using a light gray background for the drawer, as per the Figma design:

image

**Review by s-hane — APPROVED** [review]

**Pull request merged**